server/src/models/Task.ts

```diff
@@ -0,0 +1,13 @@
+class Task {
+  private timeStart: number;
+
+  constructor(public task: string) {
+    this.task = task;
+    this.timeStart = new Date().getTime();
```

Comment:
to wlasciwosc uczynilbym publiczna ale readonly i nazwalbym ja wlasnie timeStamp